src/main/java/mate/academy/dto/BookDto.java

```diff
@@ -1,14 +1,23 @@
 package mate.academy.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.UniqueElements;
 
 @Data
 public class BookDto {
     private Long id;
+    @NotNull
```

Comment:
Why do you need validation for this object? Will you apply validation for this object?

Reply:
i did mistake, i don't use this object and i don't need validation for its.